packages/paste-core/components/page-header/src/PageHeader.tsx

```diff
@@ -0,0 +1,47 @@
+import { Box, safelySpreadBoxProps } from "@twilio-paste/box";
+import type { BoxProps } from "@twilio-paste/box";
+import type { HTMLPasteProps } from "@twilio-paste/types";
+import * as React from "react";
+
+export interface PageHeaderProps extends HTMLPasteProps<"div"> {
+  children?: React.ReactNode;
+  /**
+   * Size of the Page Header
+   * @default 'default'
+   * @type {'default' | 'compact''}
+   * @memberof PageHeaderProps
+   */
+  size?: "default" | "compact";
+  /**
+   * Overrides the default element name to apply unique styles with the Customization Provider
+   * @default 'PAGE_HEADER'
+   * @type {BoxProps['element']}
+   * @memberof PageHeaderProps
+   */
+  element?: BoxProps["element"];
+}
+
+export const PageHeaderContext = React.createContext<Pick<PageHeaderProps, "size">>({} as PageHeaderProps);
```

Comment:
issue: I think because we're exporting `<PageHeaderContext />` as a component, this is potentially a really confusing name for the _react context_. We should rename one of them.

Reply:
updating to PageHeaderSetting